cairo-sys-rs/src/lib.rs

```diff
@@ -952,6 +952,13 @@ extern "C" {
     ) -> cairo_status_t;
     pub fn cairo_surface_get_reference_count(surface: *mut cairo_surface_t) -> c_uint;
     pub fn cairo_surface_mark_dirty(surface: *mut cairo_surface_t);
+    pub fn cairo_surface_mark_dirty_rectangle(
```

Comment:
Don't you want to add safe bindings for this?

Reply:
@GuillaumeGomez You think that will be problem if we add `mark_dirty` and `mark_dirty_rectangle` directly on `Surface` ?

Reply:
That could work too. Adding them then.